clogMonitor/cypress/integration/TimeRangeTest.spec.js

```diff
@@ -1,80 +1,71 @@
-      // cy.get('.timerange > #timerangestack > #startformcontrol').should("have.length", 5);
-      cy.get('.timerange > #timerangestack').contains("End Time").should("exist");
-    });
+  it("displays both start and end date", () => {
+    // to check if the dropdown is visible
+    cy.get(".timerange").should("be.visible");
 
-    it('displays what date the user enters', () => {
-      cy
-        .get('[id="startformcontrol"]').click()
-        .type('2017-06-01T08:30')
-        .get("#startimeinput")
-        .should('have.value', '2017-06-01T08:30');
-      cy
-        .get('[id="endformcontrol"]').click()
-        .type('2017-06-01T08:30')
-        .get("#endtimeinput")
-        .should('have.value', '2017-06-01T08:30');
-    });
+    cy.get(".timerange > #timerangestack")
+      .contains("Start Time")
+      .should("exist");
+    // cy.get('.timerange > #timerangestack > #startformcontrol').should("have.length", 5);
+    cy.get(".timerange > #timerangestack").contains("End Time").should("exist");
+  });
 
-    it('shows an error when the start time is after the end time', () => {
-      cy
-        .get('[id="startformcontrol"]').click()
-        .type('2018-06-01T08:30');
-      cy
-        .get('[id="endformcontrol"]').click()
-        .type('2017-06-01T08:30')
-        .contains('Start must be before End');
-    });
+  it("displays what date the user enters", () => {
+    cy.get('[id="startformcontrol"]')
+      .click()
+      .type("2017-06-01T08:30")
+      .get("#startimeinput")
+      .should("have.value", "2017-06-01T08:30");
+    cy.get('[id="endformcontrol"]')
+      .click()
+      .type("2017-06-01T08:30")
+      .get("#endtimeinput")
+      .should("have.value", "2017-06-01T08:30");
+  });
 
-    it('shows an error when improper input is entered', () => {
-      
-      cy
-        .get('[id="startformcontrol"]').click()
-        .type('0000-00-00T08:30')
-        .contains('Input is missing a value');
-      cy
-        .get('[id="endformcontrol"]').click()
-        .type('2017-00-01T08:30')
-        .contains('Input is missing a value');
-        
-    });
-
-    it("validates if there is an error, the apply button is disabled", () => {
-      cy
-        .get('[id="startformcontrol"]').click()
-        .type('2018-06-01T08:30')
-      cy
-        .get('[id="endformcontrol"]').click()
-        .type('2017-06-01T08:30')
-     
-  
-      cy.get(".apply-filters-btn").should("be.disabled");
-    });
+  it("shows an error when the start time is after the end time", () => {
+    cy.get('[id="startformcontrol"]').click().type("2018-06-01T08:30");
+    cy.get('[id="endformcontrol"]')
+      .click()
+      .type("2017-06-01T08:30")
+      .contains("Start must be before End");
+  });
 
+  it("shows an error when improper input is entered", () => {
+    cy.get('[id="startformcontrol"]')
+      .click()
+      .type("0000-00-00T08:30")
+      .contains("Input is missing a value");
```

Comment:
Here and at line 62, the error message has changed to "Input is missing a necessary value"
You can change to match exactly or you can just have "Input is missing", since both will .contain() that.